### PR TITLE
INT-2949: Added support for CommentChange event

### PR DESCRIPTION
**Pull request by tiny-james**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Added
+- Added prop for the event `CommentChange`.
+
 ### Fixed
+- Added `CommentChange` to the list of events that trigger the onEditorChange handler.
```
Comment (lnewson) [inline CHANGELOG.md:12]:
```suggestion
- Added `CommentChange` to the list of events that trigger the `onEditorChange` handler. 
```
Not sure of the normal style here, but as it's a function name I figured it should be wrapped in backticks 🤷

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Added
```
Comment (lnewson) [inline CHANGELOG.md:8]: If this is a new feature/minor, do we need to bump the package.json version as well?

**Review by lnewson — APPROVED** [review]

**Review by jscasca — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (tiny-james) [reply CHANGELOG.md:12]: Agreed.

File: CHANGELOG.md (hunk 2 above)
Comment (tiny-james) [reply CHANGELOG.md:8]: Good point.

Comment (tiny-james) [reply CHANGELOG.md:8]: done

**Pull request merged**